frontend/src/app/components/patient-info-form/patient-info-form.component.html

```diff
@@ -4,22 +4,36 @@
     <form [formGroup]="form">
       <div class="row">
         <app-form-field label="Jméno *" [invalid]="isControlInvalid('firstName')" [errors]="getControlErrors('firstName')">
-          <input formControlName="firstName" name="firstName" placeholder="Vaše jméno" required maxlength="64">
+          <input formControlName="firstName" name="firstName" placeholder="Vaše jméno" required maxlength="256">
```

Comment:
proc to? Myslim, ze @misslecter  to takhle omezovala, aby neslo zadat ultra dlouhe jmeno, protoze pak to vizualne vypadalo hnusne

Reply:
ten samý koment platí všude kdes to měnil...

Reply:
Čísla jsem vyplnil podle limitů v databázi. Myslel jsem že to není nekonzistentní schválně. Podle mě lepší když to bude vypadat hnusně než když člověk nebude moct vyplnit co potřebuje